src/main/java/com/tvtracker/enterprise/dao/BaseDAO.java

```diff
@@ -1,283 +0,0 @@
-package com.tvtracker.enterprise.dao;
-
```

Comment:
I deleted this file because this is not how we are supposed to implement a database for this course. This course specifically requires us to use SpringBoot and MySQL. The current implementation requires many many more lines of code than necessary. Rewriting the methods that CrudRepository gives us for free creates technical debt because now a "special snowflake" implementation has to be managed. Read more about technical debt here: https://martinfowler.com/bliki/TechnicalDebt.html